<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Bakery</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="4,1">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item>
     <widget class="QListWidget" name="outputsListWidget">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="verticalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="listWidgetLog">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionFileQuit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionViewZoomIn"/>
    <addaction name="actionViewZoomOut"/>
    <addaction name="actionViewResetZoom"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAboutAbout"/>
    <addaction name="actionAboutAboutQt"/>
   </widget>
   <widget class="QMenu" name="menuTask">
    <property name="title">
     <string>Task</string>
    </property>
    <addaction name="actionTaskNew"/>
    <addaction name="actionTaskRepeatLast"/>
    <addaction name="actionTaskSaveOutput"/>
    <addaction name="actionTaskTerminateAllPlugins"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTask"/>
   <addaction name="menuView"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionFileQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionViewZoomIn">
   <property name="text">
    <string>Zoom in</string>
   </property>
   <property name="shortcut">
    <string>+</string>
   </property>
  </action>
  <action name="actionViewZoomOut">
   <property name="text">
    <string>Zoom out</string>
   </property>
   <property name="shortcut">
    <string>-</string>
   </property>
  </action>
  <action name="actionViewResetZoom">
   <property name="text">
    <string>Reset zoom</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </action>
  <action name="actionAboutAboutQt">
   <property name="text">
    <string>About Qt...</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="actionTaskNew">
   <property name="text">
    <string>New...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionTaskTerminateAllPlugins">
   <property name="text">
    <string>Terminate all plugins</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionTaskSaveOutput">
   <property name="text">
    <string>Save output...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionTaskRepeatLast">
   <property name="text">
    <string>Repeat last</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionAboutAbout">
   <property name="text">
    <string>About Bakery...</string>
   </property>
   <property name="toolTip">
    <string>About Bakery</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
